### ISSUE TITLE
object-curly-newline broken for ImportDeclaration with count 1

### ISSUE THREAD

**Issue by andreineculau** [eslint/eslint#10036]
**Tell us about your environment**

* **ESLint Version:** 4.18.1
* **Node Version:** 9.6.1
* **npm Version:** 9.7.1

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
    'object-curly-newline': ['warn', {
      ObjectExpression: {
        multiline: true,
        minProperties: 1,
        consistent: true
      },
      ObjectPattern: {
        multiline: true,
        minProperties: 2,
        consistent: true
      },
      ImportDeclaration: 'always',
      ExportDeclaration: 'always'
    }],
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import {
  getCallerInfo
} from './util';
```

**What did you expect to happen?**

Nothing.

**What actually happened? Please include the actual, raw output from ESLint.**

eslint fixes the above by removing the indentation as

```js
import {
getCallerInfo
} from './util';
```

since it complains that 

```
  4:8  error  Unexpected line break after this opening brace   object-curly-newline
  6:1  error  Unexpected line break before this closing brace  object-curly-newline
```

---

related PR https://github.com/eslint/eslint/pull/9876 @nicholaschuayunzhi

**Comment by nicholaschuayunzhi** [comment]
Hi @andreineculau, I cant seem to reproduce the issue with the configuration given as the `ImportDeclaration` is set to always have a new line after the brace. 

I can get the error message by changing the configuration to this:
```js
"rules": {
   'object-curly-newline': ['warn', {
      ObjectExpression: {
        multiline: true,
        minProperties: 1,
        consistent: true
      },
      ObjectPattern: {
        multiline: true,
        minProperties: 2,
        consistent: true
      },
      ImportDeclaration: 'never',
      ExportDeclaration: 'never'
    }],
  }
```
and when i run `--fix`, the fixed code is as such:

```js
import {getCallerInfo} from './util';
```

**Comment by andreineculau** [comment]
Thanks @nicholaschuayunzhi ! 
😞 indeed the issue is on my side. [I have a second config that wants to override the rule configuration](https://github.com/tobiipro/support-firecloud/blob/master/repo/mk/js.lint.eslint.mk#L5) and I didn't set anything for ImportDeclaration nor ExportDeclaration so maybe they were conflicting.

In short, the config posted in the issue *plus this one https://github.com/tobiipro/eslint-config-firecloud/blob/master/no-ide.js * gives the error that I mentioned.

I'm closing. Sorry for the noise!
